cont3xt/vueapp/src/utils/displayValues.js

```diff
@@ -0,0 +1,37 @@
+import dr from 'defang-refang';
```

Comment:
Not sure findDisplayValue is the right name. Should it be format/display/...? Also the file name and function name should probably match if this is the only thing? Does it need its own file?

Reply:
I changed the function (and filename) to formatValue and added some more comments. I figured it would be most appropriate to have it in its own file. I suppose it could also be put in filters.js (although I didn't put it here since findDisplayValue/formatValue has more to do with how the data is actually stored by integrations).